apps/etterlatte-saksbehandling-ui/client/src/shared/types/IDetaljertBehandling.ts

```diff
@@ -28,6 +28,11 @@ export interface IDetaljertBehandling {
   kilde: Vedtaksloesning
 }
 
+export const virkningstidspunkt = (behandling: IDetaljertBehandling) => {
+  if (!behandling.virkningstidspunkt) throw new Error('Mangler virkningstidspunkt')
```

Comment:
Skjer det noe for bruker hvis det mangler virkningstidspunkt? Isåfall burde de heller gjøre en sjekk som gir bruker informasjon istedenfor å bare kaste en Error

Reply:
Dette er en sånn irriterende "dette kommer aldri til å skje men må håndtere undefined". Det er ikke mulig å komme lenger en første steg i behandling uten virkningstidspunkt

Reply:
Jeg kan jo vurdere en ! da